config/initializers/new_framework_defaults_5_2.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+# Be sure to restart your server when you modify this file.
+#
+# This file contains migration options to ease your Rails 5.2 upgrade.
+#
+# Once upgraded flip defaults one by one to migrate to the new default.
+#
+# Read the Guide for Upgrading Ruby on Rails for more info on each option.
+
+# Make Active Record use stable #cache_key alongside new #cache_version method.
+# This is needed for recyclable cache keys.
+# Rails.application.config.active_record.cache_versioning = true
```

Comment:
Should't we uncomment this and other settings in this file?

Reply:
As long as we use any caching

Reply:
I was wrong. Setting is this file is new defaults in 5.2. So, we should uncomment and check application before changing `config.load_defaults` to `5.2` in `config/application.rb`

Reply:
@eitoball Updated ded3994